app/src/shared/i18n/index.js

```diff
@@ -1,13 +1,5 @@
-// @flow
-
 import en from './en.po'
-import de from './de.po'
```

Comment:
As a side effect of this PR, I also dropped non-en translations. Seems only reasonable at this point.